lib/emoji.js

```diff
@@ -0,0 +1,22 @@
+var emojis = {
```

Comment:
Think it'd be better to have this as a JSON file :). You can require those too, as long as you include the `.json` suffix: `require('./emojis.json')`.